new/INS0984MLQM.xml

```diff
@@ -48,6 +48,7 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                     <placeName xml:lang="gez" xml:id="n1">መንበረ፡ ልዑል፡ ቅዱስ፡ ማርቆስ፡</placeName>
                     <placeName xml:lang="gez" corresp="#n1" type="normalized">Manbara Lǝʿul Qəddus Mārqos </placeName>
                     <country ref="wd:Q115">Ethiopia</country>
+                    <settlement ref="wd:Q3624">ʾAddis Ababā</settlement>
```

Comment:
sorry for nitpicking, did we not say we give internal LOC refs and not wd for places, to be able to see attestations etc.?